client/src/components/SupportTools/SupportTools.tsx

```diff
@@ -134,99 +153,157 @@ const ActiveAudits = () => {
   if (!elections.isSuccess) return null
 
   return (
-    <Column>
-      <H2>Active Audits</H2>
+    <>
+      <H3 style={{ marginBottom: '20px' }}>Active Audits</H3>
       <List>
         {elections.data.map(election => (
           <LinkItem key={election.id} to={`/support/audits/${election.id}`}>
-            <div>
-              <div style={{ color: 'black' }}>{election.organization.name}</div>
-              <div className="bp3-text-large">{election.auditName}</div>
+            <div
+              style={{
+                display: 'flex',
+                justifyContent: 'space-between',
+                alignItems: 'center',
+                width: '100%',
+              }}
+            >
+              <div>
+                <div style={{ color: 'black' }}>
+                  {election.organization.name}
+                </div>
+                <div className="bp3-text-large">{election.auditName}</div>
+              </div>
               <AuditStatusTag currentRound={election.currentRound} />
             </div>
           </LinkItem>
         ))}
       </List>
-    </Column>
+    </>
   )
 }
 
 const Organizations = () => {
   const organizations = useOrganizations()
+  const [filterText, setFilterText] = useState<string>('')
+
+  const handleInputChange = (e: ChangeEvent<HTMLInputElement>): void => {
+    setFilterText(e.target.value)
+  }
+
+  if (!organizations.isSuccess) return null
+
+  return (
+    <>
+      <H3 style={{ marginBottom: '20px' }}>Organizations</H3>
+      <input
+        className={Classes.INPUT}
+        onChange={handleInputChange}
+        placeholder="Filter organizations..."
+        style={{
+          marginBottom: '20px',
+          width: '100%',
+        }}
+        type="text"
+        value={filterText}
+      />
```

Comment:
We have a nice `FilterInput` component with a little icon in Atoms/Table.tsx. Could use if you like it